### PR TITLE
add aggregate tests

### PR DESCRIPTION
**Pull request by DavidBadura**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+help:                                                                           ## shows this help
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_\-\.]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+.PHONY: php-cs-check
+php-cs-check:                                                                   ## run cs fixer (dry-run)
+	PHP_CS_FIXER_FUTURE_MODE=1 vendor/bin/php-cs-fixer fix --allow-risky=yes --diff --dry-run
```
Comment (DanielBadura) [inline Makefile:6]: risky is part of the config, we should remove this here

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+.PHONY: php-cs-fix
+php-cs-fix:                                                                     ## run cs fixer
+	PHP_CS_FIXER_FUTURE_MODE=1 vendor/bin/php-cs-fixer fix --allow-risky=yes
```
Comment (DanielBadura) [inline Makefile:10]: same as above

File: composer.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "friendsofphp/php-cs-fixer": "^2.16.7",
     "phpstan/phpstan": "^0.12.54",
     "phpunit/phpunit": "^9.4.3",
+    "symfony/var-dumper": "^5.1",
```
Comment (DanielBadura) [inline composer.json:30]: Do we really need this?

**Review by DanielBadura — APPROVED** [review]
I think the patch is fine as it is, only some minor things

File: composer.json (hunk 3 above)
Comment (DavidBadura) [reply composer.json:30]: As a development tool, I would leave it in because I use it a lot.

**Pull request merged**